docs/en/stack/ml/df-analytics/ml-dfa-phases.asciidoc

```diff
@@ -0,0 +1,76 @@
+[role="xpack"]
+[[ml-dfa-phases]]
+=== How a {dfanalytics-job} works
+
+This page provides high-level background information on how {dfanalytics-jobs} 
+work, what kind of phases the jobs have, and what happens during the individual 
+phases.
+
+
+[discrete]
+==== The phases of a {dfanalytics-job}
+
+A {dfanalytics-job} has two states: started and stopped. Until its state 
```

Comment:
The states are actually many and in this logical order we have: starting, started, reindexing, analyzing, stopping, stopped, failed.